packages.txt

```diff
@@ -1,10 +1,10 @@
 # Essential alpine packages
-awless@cloudposse
+awless@cloudposse_amd64
```

Comment:
Some maintainers don't support arm64 packages in their releases. I had to filter them out and use x86 packages instead.